src/middleware/SessionMiddleware.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseMiddleware } from 'inversify-express-utils';
+import { Request, Response, NextFunction } from 'express';
+import { injectable, inject } from 'inversify';
+import { RedisService } from '../services/RedisService';
+import { TYPES } from '../Types';
+
+@injectable()
+export class SessionMiddleware extends BaseMiddleware {
```

Comment:
Session management isn't part of this task. Keep it small :-)

Reply:
Agree with @n-r-b

Reply:
Agreed 👍